libexec/pyenv

```
@@ -40,7 +40,7 @@ else
     local path="$1"
 
     while [ -n "$path" ]; do
-      cd "${path%/*}"
+      cd "${path%/*}" 2>/dev/null
```

Doesn't look right to be to just discard errors here.